src/constants/index.jsx

```diff
@@ -37,17 +37,17 @@ export const knownFilters = [
   'zip_code'
 ]
 
-export const SLUG_SEPARATOR            = '•'
+export const SLUG_SEPARATOR = '•'
 
 // Keyboard
-export const VK_DOWN                   = 40
-export const VK_ENTER                  = 14
-export const VK_ESCAPE                 = 27
-export const VK_RETURN                 = 13
-export const VK_TAB                    = 9
-export const VK_UP                     = 38
+export const VK_DOWN = 40
+export const VK_ENTER = 14
+export const VK_ESCAPE = 27
+export const VK_RETURN = 13
+export const VK_TAB = 9
+export const VK_UP = 38
 
-///////////////////////////////////////////////////////////////////////////////
+/// ////////////////////////////////////////////////////////////////////////////
```

Comment:
Some weirdness here adding a space on the eslint fix but nothing to necessarily change. I also liked the formatting of constants but not necessarily worth writing a very specific lint rule for that spacing. Might be worth ignoring that file, however to allow for more readable constants if we prefer that format.